scripts/word2md.ts

```diff
@@ -258,13 +261,23 @@ function convertDocumentToMarkdown(doc: Word.Document): string {
 
     function writeParagraph(p: Word.Paragraph) {
 
-        var text = p.range.text;
+        var range = p.range;
+        var text = range.text;
         var style = p.style.nameLocal;
-        var inTable = p.range.tables.count > 0;
+        var inTable = range.tables.count > 0;
         var level = 1;
         var sectionBreak = text.indexOf("\x0C") >= 0;
 
         text = trimEndFormattingMarks(text);
+        if (text === "/") {

```

Comment:
Sure, I will add a comment. An inline image shows up in the text just as a "/". When I see a paragraph consisting of nothing but "/", I check to see if the paragraph contains hidden text and if so I emit that instead. The hidden text is assumed to contain an appropriate markdown image link. I could look at automatically extracting the images, but I'm not sure how much time I want to sink into further exploring the Word object model.